R/content.R

```diff
@@ -925,3 +975,57 @@ get_content_permissions <- function(content, add_owner = TRUE) {
   res <- content$permissions(add_owner = add_owner)
   parse_connectapi_typed(res, connectapi_ptypes$permissions)
 }
+
+#' Render a content item.
+#' 
+#' @description Submit a request to render a content item. Once submitted, the
+#' server runs an asynchronous process to render the content. This might be
+#' useful if content needs to be updated after its source data has changed,
+#' especially if this doesn't happen on a regular schedule.
+#' 
+#' Only valid for rendered content (e.g., most Quarto documents, Jupyter
+#' notebooks, R Markdown reports).
+#' 
+#' @param content The content item you wish to render.
+#' @return A [VariantTask] object that can be used to track completion of the render.
+#' @export
+content_render <- function(content) {
+  suppressWarnings({
```

Comment:
Why are we blanket suppressing warnings?

Reply:
Because the variant APIs are experimental and warn the first time they're used in a session.

I'll go through and more tightly scope the warnings though.

Reply:
I think there is a `scoped_experimental_silence()` function or something that the tests use, that could be used here.

Or delete the experimental warnings. They're annoying and surely everyone just ignores them.

Reply:
> Or delete the experimental warnings. They're annoying and surely everyone just ignores them.

I will second this. I think they are so common with connectapi for doing things that are helpful to people that mot have gotten alarm fatigue and totally ignore them. Especially if scoping the suppression is tedious, better (IMHO) to just remove this one

Reply:
Scoping here was very easy, not at all tedious. I'd like to keep this as-is for this PR (I don't want to release an "official" feature that spews out two warnings — there are two without suppressing them!) and remove the warnings in a future PR. I'll create an issue to track.

Reply:
Oh @nealrichardson you're totally right, `scoped_experimental_silence()` is it.